src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto

```diff
@@ -541,6 +541,9 @@ message BidExt {
         // Right marketplace. Omitted for open auction.
         optional string pmptier = 10;
 
+        //OD15838 Temporary field to allow for transition period
```

Comment:
This is a public schema, so we shouldn't refer to our internal tickets in the comments